traits/tests/test_listeners.py

```diff
@@ -84,7 +84,7 @@ def _name_changed(self):
 
 
 class Test_Listeners(unittest.TestCase):
```

Comment:
do you also want to update the test class name?

Reply:
Grr. Not really. There's an absolute ton of little issues like this, and if we start fixing them we'll never stop and this PR will never get merged. :-(

Reply:
Done, in this case. We can do more cleanup in a future PR.